### PR TITLE
Update Slack channels

### PR DESCRIPTION
**Pull request by tashimi**
Signed-off-by: Tasha Drew tasha@chef.io

### CODE REVIEW

**Comment by thesentinels** [comment]
Thanks for the pull request! Here is what will happen next:
 1. Your PR will be reviewed by the maintainers
 2. If everything looks good, one of them will `approve` it, and your PR will be merged.

Thank you for contributing!

File: RELEASE.md [hunk 1]
```diff
@@ -56,7 +56,7 @@ Create release in [GitHub](https://github.com/habitat-sh/habitat/releases)
 On the Github releases page, there should already be a tag for the release (pushed up previously).
 Draft a new Release, specify the tag, and title it with the same (eg, 0.18.0). Then hit Publish Release.
 
-# Drink beer
+# Drink a fine glass of Cabernet Sauvignon, preferably from the Stag's Leap District
```
Comment (mwrock) [inline RELEASE.md]: This is a MAJOR improvement!

**Comment by mwrock** [comment]
need to add the DCO :)

File: RELEASE.md (hunk 1 above)
Comment (fnichol) [inline RELEASE.md]: We may have to host a Habitat Summit to discuss this. I suggest a cozy lodge somewhere in the Rocky Mountains with a roaring fire and a view of a lake.

**Comment by reset** [comment]
I'm going to close this until @tashimi gets her DCO stuff settled ;)

**Pull request closed without merging**